style.css

```diff
@@ -460,3 +463,181 @@ footer .mobile {
   border-radius: 10px;
   margin: 0 auto 8px auto;
 }
+
+/* Desktop Versin (768px) */
+
+@media (min-width: 768px) {
+  header {
+    height: 72px;
+    padding: unset;
+  }
+
+  nav {
+    font-size: 20px;
+    width: 80%;
+  }
+
+  .nav-items {
+    display: flex;
+    font-size: 15px;
+  }
+
+  nav a .fa-solid {
+    display: none;
+  }
+
+  /* Home */
+
+  .home-section {
+    background: #1c1a19 url("resources/desktop\ wallpaper.png") 100% 110% no-repeat;
+    background-size: 590px;
+  }
+
+  .home {
+    height: calc(100vh - 72px);
+    width: 80%;
+    margin: auto;
+    padding: unset;
+  }
+
+  .home-content {
+    width: 40%;
+  }
+
+  .home .intro h1 {
+    font-size: 72px;
+    margin-bottom: 24px;
+  }
+
+  .home .job h2 {
+    font-size: 32px;
+    margin-bottom: 24px;
+  }
+
+  .home .bio p {
+    font-size: 20px;
+  }
+
+  .home .socials ul {
+    flex-direction: column;
+    position: absolute;
+    left: 5%;
+    top: 50%;
+    transform: translate(0, -50%);
+  }
+
+  /* recent work */
+
+  .recent-work {
+    width: 80%;
+    margin: auto;
+  }
+
+  .recent-work .heading {
+    flex-direction: row;
+    margin: 135px 0 114px 0;
+    justify-content: flex-start;
+
+    /* gap: 24px; */
+  }
+
+  .recent-work .heading > .title {
+    padding-top: 0;
+  }
+
+  .recent-work .line {
+    margin: 0 auto;
+    min-width: 65%;
+  }
+
+  .works {
+    width: 100%;
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto auto;
+    gap: 35px;
+  }
+
+  .works .main {
+    display: flex;
+    grid-column: 1 / 4;
+  }
+
+  .works .main .poster {
+    width: 60%;
+  }
+
+  .recent-work .works .description > .title {
+    font-size: 40px;
+  }
+
+  .description > .about {
+    font-size: 20px;
+  }
```

Comment:
Wow 🔥 🔥 💪  This is an amazing webpage with simplicity at its core 👍 
- [ ] Let's check a few things to be sure we meet the requirements though:
  - [ ] Did you also notice that the `background-image`s for the project cards on mobile are not the same as the desktop view unlike yours?
  - [ ] Again, it appears some headings are dynamically changing form when in the desktop view but all seems to be same in your. Can you change this please?
  - [ ] A major observation if you examine the Figma wireframe is that whenever the user hovers over any project, the only visible element should be the action button i.e. the `background-image` becomes clearly, the `title`, `paragraph` texts and the `list` of skills become invisible.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/220817851-c724cfe6-6ecd-423c-9886-deaac949b272.png)|![image](https://user-images.githubusercontent.com/99715640/220817882-b3a5dba4-e456-4808-b254-6097132bbbc0.png)|

_**You have a beautiful page already but please, kindly work on these points to make your design match the requirement**_

Reply:
@AbayomiOlaoye  thanks for the review. I used my own work as the background image. Do I need to change?
Also, I didn't understand second instruction.

Reply:
@popey17 yes, I can tell, and that's absolutely amazing! However, since the requirement intends that some contents like `project title` and `background-image` don't stay the same on both views - desktop and mobile, I think you can get personal images or just use the ones on Figma as placeholders. This will really exude the beauty of your dextrous aesthetics even more 😄.
- By "the second", I hope I have clarified what I meant 🤔 ?

Reply:
- Let's try and have 2 project titles for both rendering, also some other pictures to use as background images in the `media-queries` when at a min-width of 768px (as you deem fit) 👍

Reply:
@AbayomiOlaoye  Thanks I will try adding different images

Reply:
@popey17 👍 Just in case you need to clarify other things or want some collaborations to get the `hover` effect that makes some content invisible, don't hesitate to contact me, please. You can also reach on Slack @Oluwatoyin Olaoye.

All the best, Chen!